src/analysis/convex/basic.lean

```diff
@@ -1108,6 +1108,23 @@ convex_hull_min (set.subset.trans hst $ subset_convex_hull t) (convex_convex_hul
 lemma convex.convex_hull_eq {s : set E} (hs : convex s) : convex_hull s = s :=
 set.subset.antisymm (convex_hull_min (set.subset.refl _) hs) (subset_convex_hull s)
 
+@[simp]
```

Comment:
_Technically_ this doesn't need to be `@[simp]`, since the lemma being `@[simp]` will also prove `convex_hull (∅ : set E) = ∅`. I don't think this is an issue though? Since the linter is happy with it.

Reply:
@shingtaklam1324 I'm not sure I understand your concern. Note that `convex.convex_hull_eq` is not marked `simp` (I guess because it requires a `convex s` hypothesis?) so there's no redundancy here.

Reply:
oh sorry, I mean the lemma below this will also prove this.